packages/devtools-source-map/src/utils/sourceMapRequests.js

```diff
@@ -0,0 +1,22 @@
+
+let sourceMapRequests = new Map();
+
+function clearSourceMaps() {
+  sourceMapRequests.clear();
+}
+
+
+function getSourceMap(generatedSourceId: string)
+    : ?Promise<SourceMapConsumer> {
+  return sourceMapRequests.get(generatedSourceId);
+}
+
+function setSourceMap(generatedId, request) {
+  sourceMapRequests.set(generatedId, request);
+}
+
+module.exports = {
+  clearSourceMaps,
+  getSourceMap,
+  setSourceMap
+};
```

Comment:
I like having the state wrapped in a module with a clear API. It gives me some more confidence that we're not manipulating the map in unintended ways. There were some other call-sites that i switched to `set` and `get` with this move.